include/crocoddyl_msgs/inertial_parameters_subscriber.h

```diff
@@ -0,0 +1,156 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020-2023, Heriot-Watt University, University of Oxford
```

Comment:
Update and fix the copyright
```suggestion
// Copyright (C) 2023-2024, Heriot-Watt University
```